config/environments/production.rb

```diff
@@ -17,4 +17,16 @@
   config.i18n.fallbacks = true
 
   config.active_support.deprecation = :notify
+  
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address:              'smtp.gmail.com',
+    port:                 587,
+    domain:               'myflix.com',
+    user_name:            ENV['gmail_username'],
+    password:             ENV['gmail_password'],
+    authentication:       'plain',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.